### ISSUE TITLE
Error: The method 'hashValues' isn't defined for the class

### ISSUE THREAD

**Issue by mark8044** [letsar/flutter_slidable#488]
It looks like latest `master` branch PR has dropped `hashList` support https://github.com/flutter/flutter/pull/151677.  This is breaking for this repo with the following errors:

```
../../../.pub-cache/hosted/pub.dev/flutter_slidable-3.1.1/lib/src/notifications_old.dart:88:23: Error: The method 'hashValues' isn't defined for the class 'SlidableRatioNotification'.
 - 'SlidableRatioNotification' is from 'package:flutter_slidable/src/notifications_old.dart' ('../../../.pub-cache/hosted/pub.dev/flutter_slidable-3.1.1/lib/src/notifications_old.dart').
Try correcting the name to the name of an existing method, or defining a method named 'hashValues'.
  int get hashCode => hashValues(tag, ratio);
                      ^^^^^^^^^^
```

**Comment by letsar** [comment]
Yes, this [PR](https://github.com/letsar/flutter_slidable/pull/484) has been merged to fixes this issue. For the moment it's only available on the master branch.

**Comment by PradeepChoran** [comment]
Waiting the update

**Comment by igorilin13** [comment]
Also ran into this, you can point to master until the new version is published:

```dart
  flutter_slidable:
    git:
      url: https://github.com/letsar/flutter_slidable.git
      ref: master
```

**Comment by PradeepChoran** [comment]
Thanks for your advice

**Comment by Shakenbeer** [comment]
> ```dart
> git:
>       url: https://github.com/letsar/flutter_slidable.git
>       ref: master
> ```

Hi there. I am a flutter noob. Where to put those lines? As far as I understand .lock file is generated...

**Comment by PradeepChoran** [comment]
dev_dependencies:
  flutter_test:
    sdk: flutter
  flutter_slidable:
    git:
      url: https://github.com/letsar/flutter_slidable.git
      ref: master

**Comment by Shakenbeer** [comment]
Thank you good sir

**Comment by JesusxAguirre** [comment]
> ```dart
> flutter_slidable:
>     git:
>       url: https://github.com/letsar/flutter_slidable.git
>       ref: master
> ```

this work for me

**Comment by yigtkaya** [comment]
> Also ran into this, you can point to master until the new version is published:
> 
> ```dart
>   flutter_slidable:
>     git:
>       url: https://github.com/letsar/flutter_slidable.git
>       ref: master
> ```

Just updated to 3.27 and got this error, this solution works

**Comment by oti-adjei** [comment]
> > Also ran into this, you can point to master until the new version is published:
> > ```dart
> >   flutter_slidable:
> >     git:
> >       url: https://github.com/letsar/flutter_slidable.git
> >       ref: master
> > ```
> 
> Just updated to 3.27 and got this error, this solution works

I'm guessing you would have to remove it from being the  direct dependency section right?

**Comment by letsar** [comment]
Sorry for the inconvenience. For some reason I didn't push the changes on pub when I merged this commit. This is done now in the 3.1.2
